docs/book/src/tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md

```diff
@@ -71,6 +167,14 @@ This section outlines considerations specific to Topology Mutation hooks:
   always return the same error message. Otherwise the system might became unstable due to controllers being overloaded
   by continuous changes to Kubernetes resources as these messages are reported as conditions. See [error messages](implement-extensions.md#error-messages).
 
+### Variable discovery guidelines
+* **Unique variable names**: When using external variable discovery, care should be taken to ensure that variable names are unique.
```

Comment:
This sentence refers to variables returned by a single call or across external patches & inline variables?

Reply:
Replaced with Fabrizio's version, but this meant across all patches initially.